src/services/utilServices/MailClient.ts

```diff
@@ -39,16 +41,11 @@ class MailClient {
 }
 export default MailClient
 
-const { NODE_ENV, POSTMAN_API_KEY } = process.env
+const NODE_ENV = config.get("env")
+const POSTMAN_API_KEY = config.get("postman.apiKey")
```

Comment:
Ok for some reason I cannot comment directly on line 58, but 
`: new MailClient(POSTMAN_API_KEY!)` can be changed to `: new MailClient(POSTMAN_API_KEY)` since we have guarentee its a `string` already

Reply:
okay done